First pass on Info (#3)
Readme-Workflows/readme-replacer: README.md

> [example]: https://github.com/Readme-Workflows/readme-replacer/blob/main/src/replacers.json

Reviewer: ```suggestion
[default]: https://github.com/Readme-Workflows/readme-replacer/blob/main/src/replacers.json
```

[discord]: https://discord.gg/2a9VC4AK6x

# Readme-Replacer

Readme-Replacer is a GitHub Action that allows you define your own collections of placeholder text to replace with specific values.  
The Action is designed to be as flexible as possible and to work with other Actions.

## Setup

To Set up the Action, first create a new workflow file and populate it with the following values:  
```yaml
name: Replace placeholders

on:
  schedule:

>     - cron: '*/30 * * * *' # Activate every 30 minutes

Reviewer: @abhijoshi2k Is this cron required, or should it trigger on push?

Reviewer 2: It depends on how user uses it. Keep it cron for now. User can change according to needs

### Replacer JSON file

A key-feature of this Action is the ability to define your own placeholders and their replacement for it.  
This is done by creating and using a JSON file (Called `customReplacer.json` and located within the `.github` directory) and configuring it to have the right values.

The structure of the JSON file may look like this:  
```json
[
  {
    "search": "{foo}",
    "replace": "Bar",
    "eval": false
  }
]
```
Here is a quick rundown of the different options:

- `search` is the text that the Action should look for in the Template file. It can be any text you like.
- `replace` is the value that the `search` value should be replaced with. In our example above would any appearance of `{foo}` be replaced with `Bar`.
- `eval` is a boolean to set whether the `replace` value should be evaluated by the Action for further manipulation. It essentially allows you to set Javascript code that would be executed when the Action runs (i.e. `new Date().toLocaleString()` would be executed as Javascript code and the result used as replacement for the search string).

> You can find an [Example JSON file][example] with some recommended placeholders in this Repository.

Reviewer: ```suggestion
You can find a [JSON file][default] which would be the default placeholders in the workflow in this repository.
```

> You can find a [JSON file][default] which would be the default placeholders in the workflow in this repository.

Reviewer: ```suggestion
You can find a [JSON file][default] which would be the default placeholders in the workflow.
```
It's not in this repository, that is default for all users, wherever and whoever uses this action will use this as the default replacers

Reviewer: Yeah, I myself forgot to remove that. Didn't notice.

Author: It's still located within this repository so it's correct.

Reviewer: > It's still located within this repository so it's correct.

I don't know what to reply for this, let's wait for @abhijoshi2k

Author: Just read the message with your changes and tell me if it makes sense.
It say you can FIND a file... where? It wouldn't mention the location.
